### ISSUE TITLE
acme subcommand generates invalid v2 acme.json

### ISSUE THREAD

**Issue by zachreizner** [containous/traefik-migration-tool#2]
I used the acme subcommand of this migration tool and then used the resulting acme-new.json with traefik v2. This would result in this error:

```
Unable to add ACME provider to the providers list: unable to get ACME account: json: cannot unmarshal array into Go value of type acme.StoredData
```

**Comment by zachreizner** [comment]
This problem seems to be caused by a disagreement between this [tool's structure](https://github.com/containous/traefik-migration-tool/blob/e8ad679ab100fe3961f23ca7ba180b61d42c2b40/acme/acme.go#L26) (`acme.StoredData{}`) of the acme.json and the [structure used in traefik v2](https://github.com/containous/traefik/blob/v2.0/pkg/provider/acme/local_store.go#L45) (`map[string]*StoredData{}`).

As an experiment to confirm this, I manually wrapped the acme-new.json from this tool with a map with a single key:
```
{ "default: <output from traefik-migration-tool> }
```
This seems to work and traefik is up and running with the migrated certificate. I haven't tested if renewal will work yet.

**Comment by ldez** [comment]
Hello, could you give me the command that you are using?

**Comment by zachreizner** [comment]
I was using `traefik-migration-tool acme` without any extra parameters. I
tried both the release binary and the master branch source but I got the
same acme-new.json from both.

**Comment by ldez** [comment]
I see your problem, I will fix that.

**Comment by ldez** [comment]
Thanks for the issue :+1: 

I fixed that in the version v0.4.0

And also with v0.4.0 you can provide a resolver name.

Note: the version will be published soon https://travis-ci.com/containous/traefik-migration-tool/builds/128305536
